engine/src/main/java/org/hibernate/search/engine/common/timing/impl/TimingSource.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.common.timing.impl;
```

Comment:
If you're going to expose `TimingSource` to other modules (e.g. the backends), you will need to put this interface in an `spi` package.